odbc/2.ibmi.md

```diff
@@ -0,0 +1,96 @@
+
+### **`odbcinst.ini`**
+
+To define your driver so that it can be referenced in your datasource, you will have to define your driver in a file called `odbcinst.ini`. This file can be found at `/etc/unixODBC/odbcinst.ini`. In this file, you define a name for each of your drivers, and tell unixODBC where it can find the driver. an example `odbcinst.ini` might look like:
+
+**`odbcinst.ini`**
+```ini
+[IBM i Access ODBC Driver]
+Description = IBM i Access ODBC Driver
+Driver      = /QOpenSys/pkgs/lib/libcwbodbc.so
+```
+(Note: You don't have to align the `=` signs, it just looks cleaner.)
+
+When you define datasources in `odbc.ini` and `.odbc.ini` (covered below), you will use the name in the `[]` brackets as the name of the driver to use.
+
+If you have multiple drivers to define, you can simply create mutiple entries in the same file, as long as each driver is headed with a driver name in brackets, and no two drivers have the same name.
+
+**`odbc.ini` and `.odbc.ini`**
+
+When you have a driver defined, you will want to also define a datasource name, or DSN. A DSN allows you to define a connection and all of your connection options in one place. When you pass a connection string to an ODBC connection, you simply need to pass the name of the DSN and all of the connection options defined with the DSN will also be included.
+
+DSNs are defined in `odbc.ini` and `.odbc.ini` (note the `.` preceding the latter). These two files have the same structure, but have one important difference: 
+
+* `odbc.ini` can be found in the IFS in `/QOpenSys/etc` and defines DSNs that are available to **all users on the system**. If there are DSNs that should be available to everyone, they can be defined and shared here
+
+* `.odbc.ini` is found in your home directory (`~/`) and defines DSNs that are available **only to you**. If you are going to define DSNs with your personal username and password, this is the place to do it.
+
+The files define DSNs similar to how `odbcinst.ini` defines drivers: You name your DSN with `[]` brackets, then specify keywords and values below them. This is an example of a  stored in `~/.odbc.ini` used to connect to an IBM i with private credentials:
+
+```ini
+[MYDSN]
+Description            = oss73dev IBM i system
+Driver                 = IBM i Access ODBC Driver
+System                 = OSS73DEV.RCH.STGLABS.IBM.COM 
+UserID                 = MYNAME
+Password               = password123
+Naming                 = 0
+DefaultLibraries       = MIRISH
+TrueAutoCommit         = 1
+```
+(**Note: The name of the driver specified in the `Driver` keyword must match the name of a driver defined in `odbcinst.ini`).
+
+Additionally, the `ibm-iaccess.rpm` will automatically create a DSN called `[*LOCAL]` in your `odbc.ini`. When using this DSN, the user credentials used will be `*CURRENT`, which is the user who is running the process that is trying to connect to the ODBC driver.
```

Comment:
The *CURRENT requires server PTFs which we should document here

Reply:
Agreed. PTFs are:

7.3: SI68113
7.2: SI69058